src/App.js

```diff
@@ -1,12 +1,10 @@
-
 import React from 'react'
+import Container from './containers/Container'
 
 export default function App() {
 	return (
 		<div>			
-			
-
-			
+			<Container />
```

Comment:
Indentation looks off here! Make sure to enable prettier following Airbnb style guid!

Reply:
Prettier is working for me and code is well indented 🤷‍♂️
![Screen Shot 2021-04-10 at 10 52 48 PM](https://user-images.githubusercontent.com/15146575/114282908-8470a200-9a4f-11eb-8c18-0a57d4fc7184.JPG)